### ISSUE TITLE
Allow user to a follow topic

### ISSUE THREAD

**Issue by bonustrack** [busyorg/busy#1332]

**Comment by pareshUD** [comment]
![recommended posts 2 2x](https://user-images.githubusercontent.com/8911074/34937795-1463e0bc-fa0c-11e7-89fd-a0f77b077bf4.png)

**Comment by jm90m** [comment]
looks good, but there should also be follow button on top of each single feed, like in here imo as well 
https://busy.org/trending/life

**Comment by pareshUD** [comment]
@jm90m Follow button is there on hover.. Check the screenshot

**Comment by jm90m** [comment]
oh i meant at the top of the feed, there should be a follow button

**Comment by Sekhmet** [comment]
Yeah, I agree. If someone looks at topic that is not in **Popular** they won't be able to follow that topic.

Maybe we could have box on the right sidebar with current topic name, description (we could give an explanation of most popular topics, what they are about) maybe in the future number of followers, statistics - number of posts daily etc, and put Follow button in this box (similar to Reddit).

**Comment by bonustrack** [comment]
How you would load multiple feed and sort them @Sekhmet ?

**Comment by Sekhmet** [comment]
bSteem is already implementing feed like this. How do you handle sorting in bSteem @jm90m?

**Comment by pareshUD** [comment]
guys what are we doing here

**Comment by jm90m** [comment]
with bSteem you are able to save a tag, and then filter based on that tag
